### PR TITLE
Credential status management

### PR DESCRIPTION
**Pull request by kezike**
This PR sets up the infrastructure to manage authorization and storage of credential status resources in source control services such as GitHub and GitLab. With this update, organizations can use their code workspace to host a repository that manages an access controlled credential status list and associated resources.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -213,56 +308,193 @@ export async function build(opts = {}) {
...
+      } catch (error: any) {
+        reply
+          .code(400)
+          .send({ message: error.message });
+      }
     }
   )
 
+  server.post(
+    '/credentials/status', async (request, reply) => {
```
Comment (kezike) [inline src/app.ts:431]: **IMPORTANT:** Although we are protecting access between the `sign-and-verify` server and the managed source control resources, we are not protecting access between the client and the server. Should we require the client to use the OAUTH token in the request?

File: package.json [hunk 2]
```diff
@@ -18,27 +17,30 @@
   "license": "MIT",
   "repository": "git@github.com:digitalcredentials/sign-and-verify.git",
   "dependencies": {
+    "@digitalbazaar/vc-status-list": "^4.0.0",
+    "@digitalbazaar/vc-status-list-context": "^3.0.1",
     "@digitalcredentials/bnid": "^2.1.2",
     "@digitalcredentials/did-method-key": "^2.0.3",
     "@digitalcredentials/ed25519-verification-key-2020": "^3.2.2",
-    "@digitalcredentials/sign-and-verify-core": "^2.0.0",
+    "@digitalcredentials/sign-and-verify-core": "github:digitalcredentials/sign-and-verify-core#revoke-credential",
```
Comment (kezike) [inline package.json:25]: **Reminder:** deploy `sign-and-verify-core` and update this line before merging

File: src/app.ts (hunk 1 above)
Comment (kezike) [reply src/app.ts]: This has been resolved in a recent commit.

File: src/app.ts [hunk 3]
```diff
@@ -213,56 +308,193 @@ export async function build(opts = {}) {
...
-      preValidation: [
-        verifyRequestDigest,
-        verifyRequestSignature
-      ]
-    },
-    async (request, reply) => {
-      const req: any = request.body;
-      const credential = req.credential;
-      const options = req.options;
+    '/issue/credentials', async (request, reply) => {
```
[the remark this answers is not in the document]
Comment (kezike) [reply src/app.ts]: This has been resolved in a recent commit.

**Review by dmitrizagidulin — APPROVED** [review]

**Comment by dmitrizagidulin** [comment]
Updated -core dependency to published latest.
Created a squashed version of this PR over at #78, for easier conflict resolution.

**Comment by kayaelle** [comment]
@dmitrizagidulin @kezike this was never merged. See list of failed checks above. Keeping in mind that we will be archiving this software, how do you want to proceed?

**Comment by kayaelle** [comment]
Discussed with team. It's been replaced with credential status manager.

**Pull request closed without merging**